✨ Add Opportunies Section (#61)
EssamWisam/cmp-docs: public/department/Extras/Opportunities/GraduateStudies.md

#### Monthly Expenses Breakdown
€300-€500 + €125 + €200-€300 + €0 for rent including all bills + health insurance + raw food + transport respectively. Transport is free for students and bills include water, gas and electricity.

You should enter the waiting list of college accommodation, but usually you wont get one and you should look for a room/apartment to rent yourself

#### Visa Necessary Condition
Being accepted at a university is not sufficient to get a visa and be able to pursue your studies in Germany. It will be necessary that you can show you will be able to *financially support yourself*. There are three possible ways for this:

I. Make what's known as a blocked account which has at least around €11400 (this guarantees to Germany that you can support yourself for the next 12 months and after 12 months they may need to ensure this again by checking your balance).

II. Get a scholarship that clearly gives you at least around 1000 euros a month for 2 years. E.g., Search for DAAD scholarships, you can also book a free appointment with them and get a consultation session from their quarters in Zamalek.

> III. Make someone in Germany demonstrate ability of covering you financially (this may be risky for them so this option only makes sense if you have a close relative in Germany).

Reviewer: ```suggestion
III. Make someone in Germany demonstrate the ability of covering you financially (this may be risky for them so this option only makes sense if you have a close relative in Germany).
```

> The most popular option is perhaps to make the blocked account and try to work during MS in Germany (part-time is allowed) so that you don't use much of that money. Understand more about blocked accounts by [watching this](https://www.youtube.com/watch?v=M9pzM-a-sOk&pp=ygUpZmludGliYSBibG9ja2VkIGFjY291bnQgbW9oYW1lZCBlbGJhc2lvbnk%3D).

Reviewer: ```suggestion
The most popular option is perhaps to make a blocked account and try to work during MS in Germany (part-time is allowed) so that you don't use much of that money. Understand more about blocked accounts by [watching this](https://www.youtube.com/watch?v=M9pzM-a-sOk&pp=ygUpZmludGliYSBibG9ja2VkIGFjY291bnQgbW9oYW1lZCBlbGJhc2lvbnk%3D).
```

For jobs, there is a special position ‘working student’ you can search through Linkedin, XING, Stepstone.

####  ✅ After MS
After your master's you are allowed to stay in Germany for 12 months to search for a job, if you get one you will change your stay from student to blue card. Alternatively, you can seek a PhD in Germany.

Learn more about lives of Egyptian students in Germany by watching [this](https://www.youtube.com/playlist?list=PLDMmn4D17zTRJoZ37--Zf55uEhwAHFedB) and videos [here](https://www.youtube.com/@MostafaKashef
).

![image](https://cloudfront.american.edu/assets/2021-images/classroom.jpg)

## 🇺🇸 Graduate Studies in the United States

> The U.S. follows a different system compared to Germany. To start off, the vast majority of MS opportunities in the U.S. are paid (and it's too expensive). Unless you get there via some external scholarship/fellowship such as [FullBright](https://fulbright-egypt.org/). For this reason, you will find that most popular opportunity in the U.S. for Egyptian students is to get a PhD in the U.S.

Reviewer: ```suggestion
The U.S. follows a different system compared to Germany. To start off, the vast majority of MS opportunities in the U.S. are paid (and it's too expensive). Unless you get there via some external scholarship/fellowship such as [FullBright](https://fulbright-egypt.org/). For this reason, you will find that the most popular opportunity in the U.S. for Egyptian students is to get a PhD in the U.S.
```

PhDs in the U.S. usually take 4-6 years and have four special properties:

#### 1. Deadlines are too early 

> For instance, to start in Fall (August) of 202X, you usually need to apply by at most 15th Dec. of 202Y where Y=X-1. This is the usual case but uncommonly, some universities support other starts than fall (e.g., spring in January) or have more relaxed deadlines. In the U.S. many people find this to be fine because it gives them an opportunity to explore the industry more instead of moving from BS to PhD without industry experience.

Reviewer: ```suggestion
For instance, to start in Fall (August) of 202X, you usually need to apply by at most 15th Dec. of 202Y where Y=X-1. This is the usual case but uncommonly, some universities support other starts not just fall (e.g., spring in January) or have more relaxed deadlines. In the U.S. many people find this to be fine because it gives them an opportunity to explore the industry more instead of moving from BS to PhD without industry experience.
```

#### 2. You need not have a MS degree to pursue it 
This is the normal case in the U.S. People that consider an MS usually consider it to be sufficient for them and don't consider a PhD. People who think they need a PhD go straight to it.

Moreover, many PhD programs are fine to award you with both a PhD and MS degree after your PhD if you fill some extra papers (without taking extra courses or doing extra research).

#### 3. They are almost always fully funded

> That is, you pay nothing; in fact, you get paid (e.g., around $2K to $4K a month depending on the location and university). Usually the more expensive the area of the university is, the higher is a stipend.

Reviewer: ```suggestion
That is, you pay nothing; in fact, you get paid (e.g., around $2K to $4K a month depending on the location and university). Usually the more expensive the area of the university is, the higher the stipend is.
```

> In return, you either do research and teaching with your PhD supervisor or just research. In the first case, it's called a Teaching Assistanship and in the latter case, it's called a Research Assistanship. Teaching duties are usually not that significant compared to Egypt.

Reviewer: ```suggestion
In return, you either do research and teaching with your PhD supervisor or just research. In the first case, it's called a Teaching Assistantship and in the latter case, it's called a Research Assistantship. Teaching duties are usually not that significant compared to Egypt.
```

Research internships aren't that common in Egypt. However, they are extremely common in the U.S. (and more common in general). **Why?** Professors in the U.S. often have their own research groups and often need help in the summer with their research. To find them you can either search for "REU" (research as undergraduate) on Google or go to the faculty section in university websites and click on a professor and see if they write on their website that they are running a research internship for their research group. In some but not all cases they may be restricted to U.S. citizens.

Likewise, you may find research internships in gulf (khaleeji) countries.

### 📝 Application Requirements

The requirements are as follows:

#### English certificate and transcript 
You easily request these from student affairs after graduation (or before for transcript). The transcript details all the courses you covered and your grades in them. You may need to submit extra material in case the transcript doesn't write the names of the chosen electives.

#### Proof of English proficiency 

> Programs usually require TOEFL or IELTS (in some cases only TOEFL and in some cases they even accept Duolingo) and it's not so common for them to just accept that your BS was in English to waive this requirement. There is a plethora of online resources to prepare to taking either of these.

Reviewer: ```suggestion
Programs usually require TOEFL or IELTS (in some cases only TOEFL and in some cases they even accept Duolingo) and it's not so common for them to just accept that your BS was in English to waive this requirement. There is a plethora of online resources to prepare for taking either of these.
```
